mysite/templates/ctms/courseunit_confirm_delete.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
```

Comment:
@alexbojko need to use the same name for all related files - not courseunit but courselet?